src/lib.rs

```diff
@@ -35,6 +35,9 @@ mod rt;
 #[cfg_attr(feature = "cargo-clippy", allow(wrong_self_convention))]
 mod auto;
 
+#[doc(hidden)]
+pub use glib; // fix to me it work in 1.28.0
```

Comment:
Or you do `pub use glib::{Cast, Continue, Error, IsA, StaticType, ToValue, Type, TypedValue, Value};`